File: 백준/Silver/14916. 거스름돈/거스름돈.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	var n, cnt int
	fmt.Fscanf(reader, "%d\n", &n)
	for true {
		if n%5 == 0 {
			cnt += int(n / 5)
			fmt.Println(cnt)
			break
		} else {
			n -= 2
			cnt += 1
		}
		if n < 0 {
			fmt.Println(-1)
			break
		}
	}
}

File: 백준/Silver/14916. 거스름돈/거스름돈.py
import sys 
input = sys.stdin.readline
sys.setrecursionlimit(10**6)
n = int(input())
cnt =0 
while True:
    if n % 5 == 0:
        cnt += (n // 5)
        print(cnt)
        break
    else:
        n -= 2
        cnt += 1
    if n < 0:
        print(-1)
        break